recipes/pngpack-py/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set version = "1.0.0" %}
+
+package:
+  name: pngpack-py
+  version: {{ version }}
+
+source:
+  url: https://github.com/axiom-data-science/pngpack/archive/v{{ version }}.zip
+  sha256: df6c6011395fb36add7c184cfbea832aef32fcee21069dec63dd27842a36ab03
+
+build:
+  number: 0
+  skip: true  # [py<35 or win]
+  script: cd pngpack-py && {{ PYTHON }} setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - libpng
+    - python
+    - cython
+    - pip
+  run:
+    - libpng
+    - python
+
+test:
+  imports:
+    - pngpack
+    - sys
```

Comment:
Do you need to import sys?

Reply:
The docs I was reading were a little unclear. I import sys in the test py file so I can exit with an error code. Feel free to modify however makes the most sense :)

Reply:
`sys` is standard library so it is expected to exists. The tests here are only for the package you are building.